src/app.ts

```diff
@@ -15,10 +15,10 @@ import log from 'electron-log'
 import electronDl from 'electron-dl'
 import electronContextMenu from 'electron-context-menu'
 
+import config from './config'
 import { init as initDebug } from './debug'
 import menu from './menu'
 import { init as initMinimalMode } from './minimal-mode'
-import WindowState from './state/window'
```

Comment:
I'm a bit concerned with moving all of the state logic into app.ts as this file will continue to grow huge and be somewhat unruly.  That is why I was trying to put all the window state logic in the separate file.

Reply:
I think we are fine for now, because it's actually much less code/logic than before. If we see this becoming an issue (`app.ts` being too complex), especially when supporting multiple windows in the future, I'd reconsider splitting this file for sure.